### ISSUE TITLE
Is it support publish/subscribe message?

### ISSUE THREAD

**Issue by prisonjun** [vcabbage/amqp#130]

**Comment by devigned** [comment]
That's what the example in the top level readme is doing: https://github.com/vcabbage/amqp#example-usage. Is there something more specific you are asking?

**Comment by prisonjun** [comment]
David, thanks for your reply!
The example is for sending/receiving messages which is using queue. I want to know if it support topic. I am using ActiveMQ.

**Comment by vcabbage** [comment]
Hi @prisonjun,

This question seems to be similar to #121.

Quoting my response to that issue:
> Topic vs queue is ultimately up to the broker, this library should already support them. Each broker handles this a bit differently. From a brief reading, AWS MQ is based on ActiveMQ. I'm not deeply familiar with ActiveMQ, but it seems that the link source/target address should be prefixed with `topic://` for ActiveMQ. Have you tried that? If that doesn't work, can you provide some additional details on what you've tried?

Can you try adding the `topic://` prefix to the link address?

**Comment by prisonjun** [comment]
Yes, you are right, it works for me now. Vcabbage, thank you very much!!

**Comment by otmezger** [comment]
@vcabbage so instead of `amqp.Dial("amqps://my-namespace.servicebus.windows.net",` I should use `amqp.Dial("topic://my-namespace.servicebus.windows.net",`? 

I tried this and got `unsupported scheme "topic"`

**Comment by vcabbage** [comment]
@otmezger The prefix should be applied to the AMQP link source or target address. Those are set with the `amqp.LinkSourceAddress` and `amqp.LinkTargetAddress` options.
